website/blog/2018-12-14-iron.md

```diff
@@ -47,12 +47,12 @@ Release notes: https://github.com/scalameta/metals-vscode/releases/tag/v1.1.0
 The extension now automatically detects a Java 8 home directory even if Java 11
 is installed on the computer. Also, the extension now works for `jenv` users.
 
-### Automatic reload window detection
+### Improved reload window detection
 
-The extension now automatically detects when "Reload window" is needed to pick
-up configuration changes.
+The extension now suggests to "Reload window" when you have an outdated "Server
+Version" setting.
```

Comment:
nit: it's not really about reload window. It can upgrade the version automatically and then the reload window feature kicks in (composability!)

Reply:
true, fixed